include/UI/UIView.h

```diff
@@ -220,16 +255,45 @@ class UIView
 
   vec2f m_histogram_point_1;
   vec2f m_histogram_point_2;
-  float m_histogram_quantiles[2]; 
+  float m_histogram_quantiles[2];
+
 
-  
   void load_ocean_color_maps();
 
   std::vector<std::string> m_ocean_color_maps_names;
 
   std::string m_color_map_directory;
 
+  float m_animation_speed;
+  std::string m_str_animation_duration;
+  
+  bool m_camera_animation_duration_open;
+  
+  std::vector<float> m_clip_maxs;
+  std::vector<float> m_clip_mins;
+
+
+  bool m_show_clock;
+  float halfScreen;
```

Comment:
Should this variable follow your notation, i.e. `m_half_screen` instead of `halfScreen`?

Reply:
removed unused variable